packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -23,10 +32,23 @@ export const useRegisterBindings = <T extends Record<string, unknown>>(
   methods?: InvokableMethods,
 ): RegisterBindingsResult<T> => {
   const testId = get(values, ["testId"]);
-  const { resolvedWidgetId, rootRef } = useWidgetId(
+  let htmlAttributes = get(values, "htmlAttributes") as Record<string, string>;
+
+  if (!isEmpty(htmlAttributes) && !isString(htmlAttributes)) {
+    htmlAttributes = mapKeys(htmlAttributes, (_, key) => key.toLowerCase());
+  }
+
+  const { resolvedWidgetId, resolvedTestId } = useWidgetId(
     id,
     isString(testId) ? String(testId) : undefined,
   );
+  const { rootRef } = useHtmlPassThrough(
+    resolvedTestId,
+    resolvedWidgetId,
+    id,
+    htmlAttributes,
```

Comment:
Take the html attributes from the values so they can be evaluated as bindings.

Reply:
```
const htmlAttributes = get(values, "htmlAttributes") as Record<
    string,
    string
  >;
  ```
  
  I take a htmlAttributes from the values using get method. are you suggesting `values.htmlAttributed` ?

Reply:
Sorry, I meant the new values, after the bindings are resolved and combined.